### PR TITLE
windows: fix build error

### PR DESCRIPTION
**Pull request by stakach**
Previously using Windows DevKit 8.1
`src\win\tcp.c(250): error C2275: 'SOCKET' : illegal use of this type as an expression`

### CODE REVIEW

**Comment by stakach** [comment]
CLA signed

File: src/win/tcp.c [hunk 1]
```diff
@@ -241,13 +241,14 @@ static int uv_tcp_try_bind(uv_tcp_t* handle,
                            unsigned int flags) {
   DWORD err;
   int r;
+  SOCKET sock;
```
Comment (saghul) [inline src/win/tcp.c]: Can you declare it inside the if (line 245) instead? It's only used there.

**Comment by saghul** [comment]
Sorry, I introduced this, my bad :-(

**Comment by stakach** [comment]
ok, should be ready to merge
Also, no worries, you do great work!

**Comment by saghul** [comment]
Landed, thanks!

**Pull request merged**